src/main/java/net/sf/rails/ui/swing/StartRoundWindow.java

```diff
@@ -178,18 +177,15 @@ public void init(StartRound round, GameUIManager parent) {
 
         players = gameUIManager.getRoot().getPlayerManager();
 
-        packet = round.getStartPacket();
-        crossIndex = new int[packet.getNumberOfItems()];
+        crossIndex = new int[round.getNumberOfStartItems()];
```

Comment:
In my tests with 1835 the changed line leads to the same result as the initial code.
But I'm not sure whether this is always the case. @neutronc do you know if both methods, i.e. `round.getStartPacket().getNumberOfItems()` should lead to the same result as `round.getNumberOfStartItems()`?